八月 10, 2020 8:08:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver resolve
信息: Remoting server accepts the following protocols: [JNLP4-connect, Ping]
八月 10, 2020 8:39:26 下午 hudson.Launcher$RemoteLaunchCallable$1 join
信息: Failed to synchronize IO streams on the channel hudson.remoting.Channel@3ce1fcfd:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769
hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@3ce1fcfd:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769": Remote call on JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769 failed. The channel is closing down or has closed down
	at hudson.remoting.Channel.call(Channel.java:991)
	at hudson.remoting.Channel.syncIO(Channel.java:1730)
	at hudson.Launcher$RemoteLaunchCallable$1.join(Launcher.java:1328)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.perform(RemoteInvocationHandler.java:931)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.call(RemoteInvocationHandler.java:905)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.call(RemoteInvocationHandler.java:857)
	at hudson.remoting.UserRequest.perform(UserRequest.java:211)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:117)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.nio.channels.ClosedChannelException
	at org.jenkinsci.remoting.protocol.NetworkLayer.onRecvClosed(NetworkLayer.java:154)
	at org.jenkinsci.remoting.protocol.impl.BIONetworkLayer.access$1800(BIONetworkLayer.java:48)
	at org.jenkinsci.remoting.protocol.impl.BIONetworkLayer$Reader.run(BIONetworkLayer.java:264)
	... 4 more

八月 10, 2020 8:39:26 下午 hudson.remoting.UserRequest perform
警告: LinkageError while performing UserRequest:UserRPCRequest:hudson.Launcher$RemoteProcess.join[](3)
java.lang.NoClassDefFoundError: hudson/util/ProcessTree
	at hudson.Proc$LocalProc.destroy(Proc.java:387)
	at hudson.Proc$LocalProc.join(Proc.java:360)
	at hudson.Launcher$RemoteLaunchCallable$1.join(Launcher.java:1321)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.perform(RemoteInvocationHandler.java:931)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.call(RemoteInvocationHandler.java:905)
	at hudson.remoting.RemoteInvocationHandler$RPCRequest.call(RemoteInvocationHandler.java:857)
	at hudson.remoting.UserRequest.perform(UserRequest.java:211)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:117)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: hudson.util.ProcessTree
	at java.net.URLClassLoader.findClass(Unknown Source)
	at hudson.remoting.RemoteClassLoader.findClass(RemoteClassLoader.java:173)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 19 more

八月 10, 2020 8:39:26 下午 hudson.remoting.Request$2 run
信息: Failed to send back a reply to the request hudson.remoting.Request$2@1aca31ca: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@3ce1fcfd:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769": channel is already closed
八月 10, 2020 8:39:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:39:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:40:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:40:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:40:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:40:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:41:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:41:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:41:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:41:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:42:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:42:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:42:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:42:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:43:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:43:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:43:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:43:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:44:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:44:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:44:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:44:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:45:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:45:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:45:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:45:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:46:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:46:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:46:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:46:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:47:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:47:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:47:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:47:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:48:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:48:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:48:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:48:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:49:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:49:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:49:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:49:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:50:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:50:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:50:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:50:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:51:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:51:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:51:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:51:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:52:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:52:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:52:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:52:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:53:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:53:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:53:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:53:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:54:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:54:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:54:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:54:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:55:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:55:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:55:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:55:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:56:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:56:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:56:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:56:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:57:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:57:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:57:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:57:56 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:58:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:58:26 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:58:41 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 8:58:51 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver resolve
信息: Remoting server accepts the following protocols: [JNLP4-connect, Ping]
八月 10, 2020 8:59:46 下午 hudson.remoting.Request$2 run
信息: Failed to send back a reply to the request hudson.remoting.Request$2@3885c870: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@743f2d89:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769": channel is already closed
八月 10, 2020 8:59:46 下午 hudson.remoting.Request$2 run
信息: Failed to send back a reply to the request hudson.remoting.Request$2@4616cfda: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@743f2d89:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769": channel is already closed
八月 10, 2020 8:59:46 下午 hudson.remoting.Request$2 run
信息: Failed to send back a reply to the request hudson.remoting.Request$2@35ee6411: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@743f2d89:JNLP4-connect connection to 192.168.0.182/192.168.0.182:44769": channel is already closed
八月 10, 2020 9:00:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:00:16 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:00:31 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:00:46 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:01:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:01:16 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:01:31 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:01:46 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:02:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:02:16 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:02:31 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:02:46 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:03:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:03:16 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:03:31 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:03:46 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:04:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:04:16 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:04:31 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:04:46 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:05:01 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver waitForReady
信息: Failed to connect to the master. Will try again: java.net.SocketTimeoutException connect timed out
八月 10, 2020 9:05:11 下午 org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver resolve
信息: Remoting server accepts the following protocols: [JNLP4-connect, Ping]
八月 10, 2020 9:05:40 下午 hudson.remoting.jnlp.GuiListener$1 run
信息: Protocol JNLP4-connect encountered an unexpected exception
java.util.concurrent.ExecutionException: org.jenkinsci.remoting.protocol.impl.ConnectionRefusalException: Connection closed before acknowledgement sent
	at org.jenkinsci.remoting.util.SettableFuture.get(SettableFuture.java:223)
	at hudson.remoting.Engine.innerRun(Engine.java:743)
	at hudson.remoting.Engine.run(Engine.java:518)
Caused by: org.jenkinsci.remoting.protocol.impl.ConnectionRefusalException: Connection closed before acknowledgement sent
	at org.jenkinsci.remoting.protocol.impl.AckFilterLayer.onRecvClosed(AckFilterLayer.java:283)
	at org.jenkinsci.remoting.protocol.FilterLayer.abort(FilterLayer.java:164)
	at org.jenkinsci.remoting.protocol.impl.AckFilterLayer.access$000(AckFilterLayer.java:45)
	at org.jenkinsci.remoting.protocol.impl.AckFilterLayer$1.run(AckFilterLayer.java:179)
	at org.jenkinsci.remoting.protocol.IOHub$DelayedRunnable.run(IOHub.java:964)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:117)
	at java.lang.Thread.run(Unknown Source)

八月 10, 2020 9:05:40 下午 hudson.remoting.jnlp.GuiListener$2 run
严重: The server rejected the connection: None of the protocols were accepted
java.lang.Exception: The server rejected the connection: None of the protocols were accepted
	at hudson.remoting.Engine.onConnectionRejected(Engine.java:828)
	at hudson.remoting.Engine.innerRun(Engine.java:768)
	at hudson.remoting.Engine.run(Engine.java:518)

八月 10, 2020 9:06:11 下午 hudson.remoting.jnlp.GuiListener$2 run
严重: Failed to connect to http://192.168.0.182:8080/tcpSlaveAgentListener/: Connection timed out: connect
java.io.IOException: Failed to connect to http://192.168.0.182:8080/tcpSlaveAgentListener/: Connection timed out: connect
	at org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver.resolve(JnlpAgentEndpointResolver.java:217)
	at hudson.remoting.Engine.innerRun(Engine.java:693)
	at hudson.remoting.Engine.run(Engine.java:518)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at sun.net.NetworkClient.doConnect(Unknown Source)
	at sun.net.www.http.HttpClient.openServer(Unknown Source)
	at sun.net.www.http.HttpClient.openServer(Unknown Source)
	at sun.net.www.http.HttpClient.<init>(Unknown Source)
	at sun.net.www.http.HttpClient.New(Unknown Source)
	at sun.net.www.http.HttpClient.New(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver.resolve(JnlpAgentEndpointResolver.java:214)
	... 2 more

